### PR TITLE
Add uforge-sync tool documentation.

### PR DESCRIPTION
**Pull request by segalaj**

### CODE REVIEW

File: end-user/en/source/pages/migration/data-synchronization.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.. Copyright 2017 FUJITSU LIMITED
+
+.. _data-synchronization:
+
+Synchronize data to the target environment
```
Comment (jgweir) [inline end-user/en/source/pages/migration/data-synchronization.rst]: As this is a title, normally the first letter is in capitals
i.e:
Synchronize Data to the Target Environment

File: end-user/en/source/pages/migration/data-synchronization.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+------------------------------------------
+
+At the end of the migration process, you can synchronize data from the source machine to the migrated instance.
```
Comment (jgweir) [inline end-user/en/source/pages/migration/data-synchronization.rst]: you might want to ensure that the migrated instance is running, so I would say:

, you can synchronize data from the source machine to the running migrated instance

File: end-user/en/source/pages/migration/scan-details.rst [hunk 3]
```diff
@@ -20,8 +20,9 @@ To view the details of a scanned Linux-based instance:
 	3. You can also filter the packages that have been modified (UForge AppCenter compares the packages scanned with its repo) by checking ``Only show the packages with changes``.
 	4. To view the more details of a package, click on the package name and then the arrow.
 
-.. note:: The number of packages between your scanned system and the one in UForge AppCenter will differ for several reasons. First, if you had more than 1 kernel only 1 is imported into UForge AppCenter. Also, UForge AppCenter adds files for install configuration and install profile.
+	.. note:: The number of packages between your scanned system and the one in UForge AppCenter will differ for several reasons. First, if you had more than 1 kernel only 1 is imported into UForge AppCenter. Also, UForge AppCenter adds files for install configuration and install profile.
 
+	5. You can also get the instructions to synchronize your data in the ``Synchronization`` sub tab (refer to :ref:`data-synchronization`).
```
Comment (jgweir) [inline end-user/en/source/pages/migration/scan-details.rst]: same comment as above, but again this is not an issue of the docs

File: admin/en/source/pages/changelog.rst [hunk 4]
```diff
@@ -3,6 +3,15 @@
...
 =========
 
+3.7.fp8
+-------
+
+Release Date: 2017-10-16
+
+New Features
+~~~~~~~~~~~~
+* Using uforge-sync binary, users can now synchronize to target environment a CentOS scan without overlay.
```
Comment (emuus) [inline admin/en/source/pages/changelog.rst]: synchronized WITH not to ... you synchronize A with B. Of course the use of "without" in this sentence makes it more complicated so I suggest ...
... users can now synchronize a CentOS scan (without overlay) with the target environment.

File: end-user/en/source/pages/migration/data-synchronization.rst (hunk 1 above)
Comment (emuus) [reply end-user/en/source/pages/migration/data-synchronization.rst]: Synchronize Data With the Target Environment

File: end-user/en/source/pages/migration/data-synchronization.rst (hunk 2 above)
Comment (emuus) [reply end-user/en/source/pages/migration/data-synchronization.rst]: you can synchronize the source machine data with the running migrated instance

File: end-user/en/source/pages/migration/data-synchronization.rst [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+.. warning:: This feature is only available for CentOS scans without overlay.
+
+From the scan detail view, you can get the instructions to synchronize your data in the ``Synchronization`` sub tab.
```
Comment (emuus) [inline end-user/en/source/pages/migration/data-synchronization.rst]: just "tab" (we don't use the term sub tab)

File: end-user/en/source/pages/migration/data-synchronization.rst [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+Download the uforge-sync tool
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You need the uforge-sync binary in the source machine, available at ``/resources/uforge-sync.bin``, e.g.:
```
Comment (emuus) [inline end-user/en/source/pages/migration/data-synchronization.rst]: e.g. should be "for example"

File: end-user/en/source/pages/migration/data-synchronization.rst [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+.. code-block:: bash
+
+    $ curl --insecure -O https://10.1.2.206/resources/uforge-sync.bin
+
+Launch the uforge-sync tool
```
Comment (emuus) [inline end-user/en/source/pages/migration/data-synchronization.rst]: as this is title put capital for Tool

File: end-user/en/source/pages/migration/data-synchronization.rst (hunk 6 above)
Comment (emuus) [reply end-user/en/source/pages/migration/data-synchronization.rst]: use double `` around uforge-sync as it is a binary name

File: end-user/en/source/pages/migration/data-synchronization.rst [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once the uforge-sync binary has been copied to the source machine, you can launch it:
```
Comment (emuus) [inline end-user/en/source/pages/migration/data-synchronization.rst]: double `` around uforge-sync

File: end-user/en/source/pages/migration/data-synchronization.rst [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+.. code-block:: bash
+
+    $ chmod +x ./uforge-sync.bin ; ./uforge-sync.bin -U http://10.1.2.206 -u guest \
+    -d 11.12.13.14 -t root -n users/guest/scannedinstances/1/scans/1 -i ~/.ssh/id_rsa
+
+There are mandatory parameters:
```
Comment (emuus) [inline end-user/en/source/pages/migration/data-synchronization.rst]: The following are mandatory ...

File: end-user/en/source/pages/migration/data-synchronization.rst [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+    * ``-U``: the UForge server URL
+    * ``-u``: the UForge user
+    * ``-d``: the migrated instance address
+    * ``-t``: the migrated instance user
+    * ``-n``: the scan URI
+.. note:: The usage of an SSH key pair is strongly recommended to connect to the migrated instance. You must precise the private key path using the ``-i`` parameter, and have the public key in the migrated instance's ``~/.ssh/authorized_keys`` file.
```
Comment (emuus) [inline end-user/en/source/pages/migration/data-synchronization.rst]: You must specify (not precise)

File: end-user/en/source/pages/migration/scan-details.rst (hunk 3 above)
Comment (emuus) [reply end-user/en/source/pages/migration/scan-details.rst]: same comment as above for me also ;)

File: admin/en/source/pages/changelog.rst [hunk 11]
```diff
@@ -3,6 +3,15 @@
...
 =========
 
+3.7.fp8
+-------
+
+Release Date: 2017-10-16
+
+New Features
+~~~~~~~~~~~~
+* Using uforge-sync binary, users can now synchronize a CentOS scan (without overlay) with the target environment.
```
Comment (cinlloc) [inline admin/en/source/pages/changelog.rst]: Just a little remark: I don't know in English, but in French when we say "Synchronize A with B", it means that A will change to be consistent with B. So when I read "Synchronize the scan with the target environment", I understand that the scan will be changed, and the target environment will stay the same and will be used as reference. What do you think as english speakers?

Comment (emuus) [reply admin/en/source/pages/changelog.rst]: yes this is what I understand also

Comment (cinlloc) [reply admin/en/source/pages/changelog.rst]: So in fact what the feature do is the opposite: it changes the target environment but not the scan. My opinion is that we should invert the two terms: "Using uforge-sync binary, users can now synchronize the target environment with a CentOS scan (without overlay)."

Comment (emuus) [reply admin/en/source/pages/changelog.rst]: agreed

**Pull request merged**